buddyboss/buddyboss-platform
# PROD-876 Memberpress signup bad nickname issue fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1946 .

### How to test the changes in this Pull Request:

1. Make sure memberpress is activated.
2. Also make sure "Members must use their email address for their Username" option is checked from MemberPress > Settings > Account page
3. Now register for a membership tier
4. See it is now preventing user to signup because email cannot not be used as nickname.

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
